corpus/filter.py

```diff
@@ -242,7 +242,7 @@ def tasks(self):
     def run(self):
         segments = []
         for seg in self.segment_file_list:
-            with open(tk.uncached_path(seg)) as f:
+            with uopen(tk.uncached_path(seg), "rt") as f:
```

Comment:
Just plugging in `seg` should be enough since `tk.uncached_path` is [called internally](https://github.com/rwth-i6/i6_core/blob/77b765eaa112924d131c4a71be374b6339d8e0cb/util.py#L148)
```suggestion
            with uopen(seg, "rt") as f:
```